model/src/main/proto/survey.proto

```diff
@@ -47,31 +51,33 @@ enum QuestionType{
 }
 
 // Corresponds to the semantic name given to each question.
-enum QuestionName{
+enum QuestionName {
   // The question name is unknown.
-   QUESTION_NAME_UNSPECIFIED = 0;
+  QUESTION_NAME_UNSPECIFIED = 0;
+
   // Corresponds to the name of the user type question.
-  QUESTION_USER_TYPE = 1;
+  USER_TYPE = 1;
 
   //  Corresponds to the market fit question.
-  QUESTION_MARKET_FIT = 2;
+  MARKET_FIT = 2;
 
   // Corresponds to  the NPS question.
-  QUESTION_NPS = 3;
+  NPS = 3;
 
   // Corresponds to  the open-ended question for a NPS promoter.
-  QUESTION_PROMOTER_FEEDBACK = 4;
+  PROMOTER_FEEDBACK = 4;
 
   // Corresponds to  the open-ended question for a NPS passive.
-  QUESTION_PASSIVE_FEEDBACK = 5;
+  PASSIVE_FEEDBACK = 5;
 
   // Corresponds to  the open-ended question for a NPS detractor.
-  QUESTION_DETRACTOR_FEEDBACK = 6;
+  DETRACTOR_FEEDBACK = 6;
 }
 
 // Represents the multiple choice answer available for a question.
-message SurveyQuestionAnswer{
-  // The ID of this answer option.
+message SurveyAnswerOption {
+  // The ID of this answer option. It is a string value that uniquely identifies the answer option
```

Comment:
Is it unique to the survey instance, profile, device installation, or global? Ditto elsewhere for new properties where we need to guarantee some level of uniqueness.

Reply:
This was addressed in the latest changes.